Update tutorials for v0.6.0.0

goobertron opened the pull request:
I've just noticed that the style class 'click' doesn't appear in bold blue on the current project site, nor do list items appear in grey, as they used to.

Unless there was a reason for removing these, this PR reintroduces them.

goobertron commented:
Thinking about this again, I'm going to try to update the tutorials fully for v0.6 before its release, so you can leave this PR for now and I'll add to it soon.

jhass commented:
:+1: for `.click`, but I'm not sure if the list items are actually better in grey.

goobertron commented:
Sure, it's a design decision. I *think* that's how it was styled previously, unless I only ever changed the CSS in my local copy (I don't think so).

I quite like the grey for lists, as it differentiates them well from the surrounding text; but I appreciate that it's a bit of a 'lowlight' as well. Let's bring some of the designers and CSS people, @svbergerem and @Flaburgan (have I missed anyone) to see what they think.

I can easily change this back (or change it to something different) as I work on updating the rest of the tutorials pages.

svbergerem commented:
I prefer the version with `.click` but without the list style changes. (I only tried that manually on the current dfo.org website since I didn't want to setup mysql to test this)

jhass commented on config/locales/en.yml:
```diff
@@ -422,18 +430,35 @@ en:
-      format_link_text: 'this tutorial'
+      format_ex1: 'There’s a lot you can do to make your text more varied: bold, italics, large headings and so on.'
+	  format_ex2: 'To add <em>italic</em>, <strong>bold</strong>, or code, highlight the text you want to format and click the relevant button from the row above the publisher window. The Markdown code for this formatting will be placed around this portion of text.'
+	  format_ex3: 'To create a heading, list or quotation, place your cursor on the line you want to format, and press the relevant button.'
```
Please don't indent with tabs, only spaces  :)

goobertron replied:
Thanks. Don't know how that one slipped through.

You can ignore this branch for now. I'll use it to commit updates to make sure they aren't lost on my machine (already happened once today), and will shout 'help' when I'm ready for review. Appreciate your attention, but don't want to waste your time until then.

jhass commented on config/locales/en.yml:
```diff
@@ -397,7 +397,18 @@ en:
+	  top_buttons_list7: 'Numbered list'
+	  top_buttons_list8: 'Code'
+	  top_buttons_list9: 'Quoted text'
+	  aspect_selector: 'Aspect selector button'
```
There are a whole bunch more, please check the whole file :)

goobertron replied:
Thanks.

Flaburgan commented on config/locales/en.yml:
```diff
@@ -422,18 +430,36 @@ en:
+      preview_ex1: 'Between the aspects and share buttons is the last button – and a really useful one! This allows you to see what your post will look like, so you can be sure you’re happy with it before you actually post it. This can really help with correcting errors, and especially with text formatting. More on this below.'
+      preview_ex2: 'That is all there is to know about the buttons and icons on the publisher. The real magic happens within the publisher itself. As well as plain text, you can add #tags, @mentions, links, format your text and embed photos, videos and audio. Wow!'
+      
+      You can also contain your link within your text by using Markdown. You can use the <span class="click">Preview</span> button to check that your links are working properly before you post your message.
```
This line is outside the `'`

Flaburgan commented:
On the `/tutorials` page, I would replace _"Find your pod on Pod Uptime to see which version of the software it is running."_ by something like "Check on the statistics page of your pod (available in the footer) which version it is running" to avoid an external dependency which is not managed by the core team.

The `/getting_started/sign_up` page still shows old screenshots (the registration form with the green background, the `/getting_started` page and the edit profile one), could you please update this screenshot too? You can go at /getting_started without creating a new account ;)

_"If there is no “sign up” link, it might be that this particular pod is not currently accepting new registrations. In this case, you’ll need to choose another pod."_ Here again we can redirect to the /statistics page to allow the visitor to check if the registrations are open or not.

On the "complete your profile" section, diaspora* 0.6 now allows the user to make the private fields visibility public (see "My extended profile"). I guess it would be nice to mention this feature here, useful to use diaspora* being an organization, a company, a project or to use it as a blog.

(Review of Part 2 and the following coming soon)

Flaburgan commented:
`/getting_started/interface`:

_"Therefore, depending on which pod you’re on, the interface might look very slightly different to how it’s described here. However, the essential elements should all be present, even if they are arranged slightly differently."_ Repetition of slightly here. This is only a style remark.

The screenshot is showing the header bar with "my aspects" in it. This is a customization of diaspora-fr, even if I plan to add it (https://github.com/diaspora/diaspora/pull/6231) it will not be part of 0.6. I don't know if it's worth redoing it. What others think?

_"1. your pod’s ID (...)"_ -> that's the pod name, not really an ID.

_"The stream is your home page in diaspora*. It aggregates all the posts available to you. That is:
1. posts made by your contacts;
2. public posts made containing #tags you follow;
3. posts that @mention you."_

It also contains posts from community spotlight, if activated.
I never heard about mentions. It would be logical to find posts which mention you inside your stream even if you don't share with the person who wrote them, but I'd like a confirmation here @jhass @supertux88 and everyone who looked at the evil query :p 

_"How much of a person’s profile information and posts are made available to you depends on your connection with them"_ Not sure what you mean here.

Another style remark, you have a repetition of "Underneath" at the bottom of the page.

You're not talking about the "My activity", "@Mentions" and "My aspects" streams here. Looks like My aspects is the next part. If you talk about those streams later, I think it's worth adding a link to the page. If you're not talking about them, then one sentence can be enough: "The @mentions stream contains all the posts where you were mentioned, order by time of creation". "The My activity stream contains all the posts you interacted with (like, comment, reshare) order by time of last activity by anybody on the post."

Flaburgan commented:
`/getting_started/aspects`:

The pop up screenshot to create a new aspect is not up to date but as nothing changed there, I would let it as it is.

_"For this exercise, I suggest creating an aspect called “Diaspora,” into which you can place people you meet here."_ IIRC there is no other place where "I" is used, this feels inconsistent. The complete sentence can be removed in my opinion.

_"However, you should use this with care, and consider other people’s privacy before doing so."_ I see no other users information leaked in this feature. This sentence can be removed imo.

goobertron commented:
Thanks Fla for all your comments. I've made the changes you have noted (including the style comments, thank you!) and created some new images. I've just pushed these changes.

goobertron commented:
> You're not talking about the "My activity", "@Mentions" and "My aspects" streams here.

Those lines were deleted accidentally in the commit which has been made live on the site - I restored them in the first commit in this PR.

goobertron commented:
> I never heard about mentions. It would be logical to find posts which mention you inside your stream even if you don't share with the person who wrote them, but I'd like a confirmation here

I've just confirmed that @mentions _do_ appear in the stream. I created a new account on a remote pod, shared with my existing account (but didn't share back from my existing account), mentioned the existing account, the post appeared in my stream.

goobertron commented:
> @jhass I'm not sure if the list items are actually better in grey.
> @svbergerem I prefer the version with .click but without the list style changes.

I've removed the grey from the list style.

goobertron commented:
There currently isn't any mention of the chat feature in the tutorials. My understanding is that this is considered not stable and isn't yet in the main code-base, so it's probably not appropriate to detail how it works.

I think it might, however, be useful to mention that it exists on some pods, with a caveat about its current status. I suggest adding it to the foot of the Interface section, saying something like:

> Some pods also run a real-time chat feature. If chat is enabled on your pod, moving your cursor to the right-hand side of one of the main stream views will reveal a gray bar. Clicking this bar will open it to reveal the chat interface.
> Note: This feature is currently experimental, so we won’t discuss it fully in these tutorials. You can find out more about it here.

What link should I use at the end?

Any comments and/or edits welcome. I have never used the chat feature, so I'm somewhat flying blind.

Flaburgan commented:
>What link should I use at the end?

When the chat is activated on a pod, the corresponding help section is available for the users. So you can simply say "check the help of your pod". More globally, I feel like the tutorials are nice for a big picture of what's diaspora* and getting started, and it's the role of the help to detail how everything in particular is working.

goobertron commented:
One further question: how best to take account of people who use the mobile interface (or the desktop version on a small screen)?

I've added a few screenshots and a bit of explanation to the Interface section (part 2), but I could create a separate Interface section for mobile users, with a link at the top of the original interface section. That way desktop users wouldn't have to get through stuff for mobile users, and vice versa. Before I start on this, what do you all think?

svbergerem commented:
> I could create a separate Interface section for mobile users, with a link at the top of the original interface section

I wouldn't mind having a separate mobile interface section if you are willing to work on that. :)

goobertron commented:
OK no problem. Will do!

goobertron commented:
OK, I think this is now ready for review. I've done everything I can think of to:

- make the tutorials up to date for 0.6.0.0
- make the instructions as simple as possible
- take account of mobile as well as desktop users.

Please have a look and let me know where things can be improved, or if you spot any errors. Once that's done, I'll rebase and squash commits.

Pinging @Flaburgan @svbergerem @denschub @jhass particularly.
Thanks!

Flaburgan commented:
Thank you @goobertron I'll see if I'll find some time to review your work, quite busy until next monday unfortunately.

goobertron commented:
Any chance of getting this reviewed and published? I'd like to get the project site up to date with the new UI as soon as possible.

Flaburgan commented:
I'm installing it right now. Sorry for the late review. (And you pinged at the right time, it's the lunchtime break!)

goobertron commented:
Thanks. Look fwd to your review.

Flaburgan commented:
**diaspora* tutorials**

Nothing to say :)

**Part 1 Sign up**:

>“seeds,”  (...) “pods.” 

The `,` and the `.` are misplaced here, please put them outside of the quote.

>1. Enter the name you want displayed on screen in the first field.
2. Click the button to upload a profile image (an “avatar”) from your computer.
3. You can pull in the public profile information from your Facebook account, if you have one.
4. (...)

The number in the list doesn't refer to anything here. Maybe you want to add it to the image file, but if you don't want to, can you please order them the same way the user meets them? This would mean put Facebook info first. As we don't really want to encourage that because of privacy issue, maybe you can add a `(Optional)` at the beginning of this one?

*To be continued...*

goobertron commented:
Thanks for your comments, @Flaburgan.

>> “seeds,” (...) “pods.”
>
> The , and the . are misplaced here, please put them outside of the quote.

Actually, that is correct punctuation style for US English, which is used far more widely than UK English across the globe, so I think is the appropriate style to use on the project site. I used to do this stuff professionally. ;-)

> This would mean put Facebook info first. As we don't really want to encourage that because of privacy issue

If we don't want to encourage it, why include it in the getting started pane at all? Anyway, that's another matter, and I agree the order is slightly odd. I'll look at improving that now.

goobertron commented:
How about:

1. Enter the name you want displayed on screen in the first field.
2. Click the button to upload a profile image (an “avatar”) from your computer.  
    Note: You can pull in the public profile information from your Facebook account, if you have one.
3. Add a few words about yourself. It might be things that interest you – perhaps music, activism – and perhaps the town you live in. These will be converted into #tags, which help you find content and help other people to discover you. We’ll explain more about #tags later in this tutorial.
4. Click Take me to diaspora* >>

Flaburgan commented:
**Part 2 – The interface**

>a link to your profile page (see Part 1);

You're talking about the profile page itself at the end of this page. The part 1 was about the edition in the setting. I guess it would be better to instead of Part 1, link (or only indicate) to the bottom of the page.

Your screenshot doesn't have the "Be excellent to each other!" but I don't think it's worth redoing it.

>On the left you’ll see links to the different content views available in diaspora*: My activity, Stream, @Mentions, My aspects, #Followed tags and Public activity.

Can you please switch "My activity" and "Stream" so the streams are listed in the same order as on the screen?

>connect to external services such as Facebook, Twitter and Wordpress

You can add Tumblr to the list ;)

*About My activity:*
>Posts in this stream are ordered according to the latest activity (comment or like) on each post.

Actually this just changed with the coming 0.6.1.0 release, it's only the last comment which count now (pinging @svbergerem to confirm)

*About the tags stream:*
>Like “My aspects”, you can filter this stream to show posts containing one or more tags by selecting and deselecting them from the tag list in the menu

That's not true. Clicking on a tag there will redirect you to the tag page.

>Original white background: the default theme with a white background. This looks more like the “old-school” diaspora* interface

It's not the default theme, maybe you can describe it as the "pre-0.6" theme.

Flaburgan commented:
>Note: You can pull in the public profile information from your Facebook account, if you have one.

Well I think the link also pull you username in addition to your avatar, that's why the link is above everything. Maybe just skip facebook at all here?

goobertron commented:
Thanks again.

> You're talking about the profile page itself at the end of this page. The part 1 was about the edition in the setting. I guess it would be better to instead of Part 1, link (or only indicate) to the bottom of the page.

I mention it on this page as part of the UI, but it is discussed in greater detail (with screenshots) at the foot of Part 1.

> Posts in this stream are ordered according to the latest activity (comment or like) on each post.
>
> Actually this just changed with the coming 0.6.1.0 release, it's only the last comment which count now (pinging @svbergerem to confirm)

You're right. Have removed '(comment or like)' so that it's correct now and will be after 0.6.1.0 is released.

> Clicking on a tag there will redirect you to the tag page.

Oops. Not sure how that made it in there. A hang-over from the original 2013 tutorials. Now fixed.

> It's not the default theme, maybe you can describe it as the "pre-0.6" theme.

No, it's the default theme, but with a white (instead of grey) background. Have added 'but' to make it clearer.

All other points accepted and changed.

> Maybe just skip facebook at all here?

Yes, good idea. Anyone who wants to do that can work it out.

goobertron commented:
NB: I've just been through and found a few examples where I had placed a punctuation mark outside the closing quote mark (UK style) - have changed these now, so no need to mention them if you find them.

Flaburgan commented:
>I mention it on this page as part of the UI, but it is discussed in greater detail (with screenshots) at the foot of Part 1.

Well as far as I see the Part 1 only talks about the `/profile/edit` page, when the bottom of part 2 describe the profile page itself. Am I up-to-date? Did I miss something?

Flaburgan commented:
>have changed these now, so no need to mention them if you find them

Don't hesitate to push each time you change anything, I'll see the commits coming here and refresh my local files :)

goobertron commented:
> Well as far as I see the Part 1 only talks about the /profile/edit page, when the bottom of part 2 describe the profile page itself.

Fair point. Have changed. Pushing in a minute.

Flaburgan commented:
**Part 3 – Aspects**

Globally, I find the order used in this section a bit odd. The user is on the "My aspects" stream, he creates an aspect so he is redirected to the contacts page, then he's told to go back to my aspects, and then to go back to contacts... Maybe after introducing the concept of Aspects you should take him directly to the contact page, make him add some contacts and then show him the stream?

>Note: if you’re using the mobile interface, you’ll first need to click Toggle mobile in the drawer in order to be able to add a new aspect.

Maybe you can add some explanation here like "click Toggle mobile in the drawer *to switch temporarly to the desktop version* to be able to add a new aspect *as this feature is not available at the moment in the mobile version*"

>Click My aspects in the menu and your list of aspects will appear. 

Here too you can be a little more precise, like "**When you're on the stream page**, click My aspects in the **left** menu (...)"

>No one else will ever be able to see the names of your aspects, and your contacts will never know which aspect(s) they are in.

This sentence looks really important to me, what about put it in bold?

>3 When you have finished, click the Create button. The aspect is then created and selected in the menu.

I would precise "When you have finished, click the Create button. *You will be redirected to the "contacts" page.* The aspect is then created and selected in the menu."

>Once the aspect has been created, you’ll be taken to a list of contacts in that aspect (which will be no one so far). There’s a link to “add contacts to Diaspora” (or whatever you have called your new aspect).

This page changed. The aspect is not presented as empty but list every contacts, those in the aspect and those not present. So at least diaspora* HQ will probably be listed here, that means no links to "add contacts".

goobertron commented:
> Globally, I find the order used in this section a bit odd.

Excellent point, and thanks for noticing that. I thought procedurally it made sense first to add an aspect and then to do other things with it, but I have reordered to explain the aspect list first, then add an aspect, then (when you are redirected to the contact page) explain the contacts page. I think it works better now.

Have implemented all your other suggestions, and made a few other changes. Have just pushed the revised version.

ps: did you check through the mobile interface page? Click the link in the first paragraph of the interface page to get to it. The text is mostly the same, with some parts missed out if they are not possible in mobile, and with different screenshots.

Flaburgan commented:
Perfect corrections to the part 3. Thank you, you're really doing a great work!

I saw the new mobile page, I think I'll review it at the end.

goobertron commented:
@Flaburgan, I'm going away for a week this Saturday, and am not planning to get online while I'm away. If, therefore, it's possible to review the rest of this PR by Friday afternoon, that would be great and I'll do my best to make all changes by Friday evening so that this can hopefully be merged without having to wait a week while I'm away. Hope that's all right.

Flaburgan commented:
I'm sorry to take so long to review this. As the content of this PR looks already better than what's actually in production, maybe we can merge it now and I'll open a issue listing my feedback? Allowing you to open another PR after that.

Flaburgan commented:
@denschub what do you think? You can merge it if you agree.

goobertron commented:
Let's get it finished before merging. It would be good to have up-to-date information on the site as soon as possible, but it's not so desperate that we need to upload it before it's finished.

I don't know whether anyone else would like to help review this?

Flaburgan commented:
About part 4. I would not have used the "following" term there but present diaspora* the way it technically works. I have to think more about it though so let's skip it at the moment and go to part 5.

**Part 5 - Start sharing!**

I don't know if this is intended but your screenshot of the publisher show the tabs with the icons instead of the text (this should only happens on small screens). It doesn't bother me you can keep it like that.

_____

>Text formatting

Please order the list with the same order than in the publisher: Bold goes first before Italic.
Maybe add a small sentence to describe the usage of the buttons, even if it looks obvious, that's why a tuto does :) So something like _"Try it by entering some text in the publisher, select it and click the Bold button. You should now see two ** at the beginning and at the end of your text. Click the preview tab to look at how your text will be rendered. Bold, right?"_
_____

>If, on the other hand, you want to announce something to the whole world, select “Public,” and there will be no restrictions on who can read your message.

We should add a precision here: Public means "The whole Internet", not "Public in diaspora*". So the search engines will probably scrawl that post and people outside diaspora* will be able to see it.
_____

>It also provides a link to an atom feed of your posts, which might be useful if you know what this means!

"If you know what this means" sounds weird. If you don't want to explain Atom and RSS here, just explain "It is useful to share your public posts outside diaspora*", but don't tease without explaining :p
________
You're talking about the locator and the image upload, but not about the poll. Maybe have a quick sentence on it? No need to detail everything, it's quite obvious, but just mention it ;)
______

The mention selector screenshot is not up-to-date with 0.6. It now contains the diaspora* ID as well. Not sure it's worth redoing it though.
_____

You have a "<p>Post preview</p>" after "Embedding" which should probably not be here.
______
The screenshot of connected services is outdated as well.
____
Ignore a user: "they will still be able to comment on, like or reshare your posts"

Actually, I think they can't. 
____
I would add somewhere a mention about the timeago being a link to the SPV. That's useful for both people to see the post in that view if they prefer, or to share the post outside of diaspora*.

Thanks again for your work!

Flaburgan commented:
**Part 6 - Notifications and conversation**

>Clicking the notifications icon (which looks like a satellite) in the header bar brings down a drop-down list showing the five most recent notifications.

It's now a bell, and the drop-down is scrollable so it's not only five most recent anymore :)
___________
>Click post to view that post.

We now display the beginning of the post so the word "post" itself doesn't appear anymore.
_____________
>In the menu, you’ll see a list of conversations, with the title, avatar of the person who started the conversation

Actually it's the avatar of the last user who answered, excluding you.
_____
>Move your cursor over ~~this bar~~ to see who else is involved.

This conversation item globally, that works.

Flaburgan commented:
**Part 7 – Finishing up**

Nothing to say here! Congrats!

Flaburgan commented:
**Formatting text**

>To create a code block, start your text/code with four backticks

Actually three are enough.
____
There are some markdown possibilities which are not described there such as tables. I guess this can be done later, and @svbergerem can help us list them.

goobertron commented:
**Part 5 - Start sharing!**

> I don't know if this is intended but your screenshot of the publisher show the tabs with the icons instead of the text (this should only happens on small screens).

I hadn't noticed that, but actually I think it's better (by accident), as it won't matter what language someone is reading the tutorial in because the icons won't need translating.

> Maybe add a small sentence to describe the usage of the buttons, even if it looks obvious, that's why a tuto does

Have added the following to the paragraph further down which starts 'To make a public post...'
"*You could try some formatting: highlight a word and click the* ***bold*** *button. You should see two asterisks appear on each side of the word, and in preview mode it will appear in bold.*"

> The mention selector screenshot is not up-to-date with 0.6.

I know. It is worth redoing, but it'll be too difficult for me to do at the moment, because I'd have to set up some dummy accounts again and I don't have diaspora working locally at the moment. I'll fix this screenshot in a later PR, or someone else could provide an up-to-date one.

> The screenshot of connected services is outdated as well.

I no longer have accounts with most of those services so can't set it up for the icons to appear for me. It'll need someone else to provide this.

>> Ignore a user: "they will still be able to comment on, like or reshare your posts"

> Actually, I think they can't. 

Really? It used to be the case, and it was the cause of much complaint from users. If you can find out for sure if and where this was fixed, I'll change the text.

> I would add somewhere a mention about the timeago being a link to the SPV.

There's a heading and short paragraph about this in Part 2 - the interface.

All other changes implemented - thanks.

goobertron commented:
> It's now a bell, and the drop-down is scrollable so it's not only five most recent anymore :)
> We now display the beginning of the post so the word "post" itself doesn't appear anymore.
> Actually it's the avatar of the last user who answered, excluding you.

Well spotted! (x3)

>> Move your cursor over this bar to see who else is involved.
>
> This conversation item globally, that works.

I'm afraid I don't understand this - could you explain more?

goobertron commented:
> **Formatting text**

I think it might make sense for someone who really understands what can be done with markdown to take over updating that page.

Flaburgan commented:
>

>>>Move your cursor over this bar to see who else is involved.

>>This conversation item globally, that works.

>I'm afraid I don't understand this - could you explain more?

Put your mouse over any part of the conversation item will make the other participants appear, not necessarily the gray bar.

goobertron commented:
Brilliant, thanks.

goobertron commented:
I believe this is now ready for merging, if no one else has any comments.

@denschub, if you're happy with it I'll squish the commits for you.

goobertron commented:
@Flaburgan, are you able to compete your review? My editing fingers are itching...

Flaburgan commented:
I just reviewed the part 2 for mobile interface and found nothing special. Only the part 4 left!

Flaburgan commented:
**Part 4 - Finding and connecting with people**

>On diaspora* there are three types of relationships between users:

I would reorganize this part. Its content is mostly fine, but it should not be split in _following, followers_ and _mutual sharing_  in my opinion. I don't feel like there are 3 types of relationships. There is only one in my opinion: sharing. And of course, depending of if the other person is also sharing, some features are activated or not. So we could keep those 3 parts, but say "You are sharing, somebody is sharing with you, you're sharing together" or something like that.

To present the relationship in diaspora*, I usually base my explanation on a comparison with what people knows: Facebook and Twitter.

So I would have modified the text after *Let’s try to make sense of what that means.* by talking about:
- Sharing in diaspora* means "I give you access to the content I'm posting".
- It is the opposite of Twitter, where you say "I'm interested about the content your posting".
- On Facebook, the relationship is equivalent, the two friends are seeing the other one the same way. On diaspora*, nothing force the user to share back with you if he doesn't want to. Don't be afraid then if you see "Somebody started sharing with you" in your notification and you don't know this person. 
- When you share with someone, you show interest for this person. Diaspora* smartly understand that and so is adding his public post to your stream and aspect streams in which you put him. This can be seen as a "follow" action, you ask to receive post from this user. However nothing force him to add you back so you will maybe ever see only his public posts.

I don't know what others think here and how to formulate that exactly, but I guess this resume my vision of the relationship in diaspora*. Of course, we still need to mention important information like _There’s no way to tell which aspect someone else has placed you in, for privacy reasons – it’s their business which of their aspects they place each person in, so no one else can find out._, or the fact that you can only have a private conversation with someone you are sharing with and who is sharing back.

**Finding people and adding them to aspects**

>As soon as your friend accepts the invite, they will be taken through the same registration process you have just been through.

I would add something like "Besides, diaspora* will propose them to share with you during this process".

>Public posts containing those tags will then appear in your stream, and you can follow people who post things which you find interesting, by placing them into one of your aspects.

You can add "You can also simply go on the page of the tag which interests you: the person who added this tag to their profile are listed in the left column. Maybe you want to share with them."

goobertron commented:
Thanks again for your detailed review. I've made the following changes:

1: Changed the three headings from 'Following', 'Followers' and 'Mutual sharing' to 'Someone shares with me', 'I share with someone' and 'We share with each other'.

2: Added the following under the main 'Sharing' heading (new part in italics):

> We call connecting with someone “sharing” because it’s an indication that you want to share content with them. _Sharing with someone in diaspora* means giving that person access to the content you post. This is the opposite of Twitter, where following someone gives **you** access to the content **they** post._

3: Reworded the opening two paragraphs under 'We share with each other' to:

> If two of you have started sharing with each other, things get more interesting, and more complicated! The two of you have each indicated a desire to see posts from the other, so each of you will see limited posts made by the other person to the aspect(s) into which the other has placed you.
> This can be compared to being “friends” on Facebook, although there are important differences. On Facebook, the relationship is equivalent, with both people getting the same access to each other’s content. On diaspora*, however, your sharing with each other might be very different. Think of the following scenario:

4: Added the following sentence to the paragraph under 'Sending an invitation email':

> They will be given the chance to share with you during this process.

5: Added the following to the end of the 'Following tags' paragraph:

> You can also click on a tag which interests you to view the stream of all posts available to you containing that tag, and you can find people you want to connect with.

I think the point that someone doesn't need to follow you back is already covered.

goobertron commented:
Have just pushed a new commit, but I've broken the translations .... nooooo!

goobertron commented:
OK, final commit pushed: I think this is now ready for merging! - unless anyone else has any comments.

goobertron commented:
@denschub, are you happy with this?

goobertron commented:
@denschub ping - is there anything further I need to do to this to make it acceptable for merging?

denschub commented:
Sorry for the delay, merged, thanks.

Flaburgan commented:
Thank you very much @gooberton for your very valuable work! And @denschub for the merge and server update ;)

goobertron commented:
Thanks, Dennis. And thanks, Fla, for your reviews.
